deploy/csv-templates/crds/ocs.openshift.io_storageclusters_crd.yaml

```diff
@@ -3562,7 +3570,7 @@ spec:
                   to distribute its data replicas for the default CephBlockPool
                 type: string
               images:
-                description: Images holds the image reconclie status for all images
+                description: Images holds the image reconcile status for all images
```

Comment:
I am wondering why this wasn't covered in earlier commits? Did we forget to run csv generation and commit? 🤔

Reply:
can we have this part as a separate commit?

Reply:
Ah, odd. And yes, I'll separate that from the main commit.

Reply:
This has been merged as #958 meanwhile.